openfisca_france/model/prelevements_obligatoires/prelevements_sociaux/cotisations_sociales/exonerations.py

```diff
@@ -5,8 +5,10 @@
 
 from numpy import datetime64, maximum as max_, minimum as min_, round as round_, timedelta64
 
-from ....base import *  # noqa analysis:ignore
-from .base import apply_bareme_for_relevant_type_sal
+from openfisca_france.model.base import *  # noqa analysis:ignore
+from openfisca_france.model.prelevements_obligatoires.prelevements_sociaux.contributions_sociales.base import montant_csg_crds
```

Comment:
The first line is the one that imports "all what you need to write a formula".

The second one, in my opinion, is something we would ideally like to avoid: direct python dependencies in the code are kind of messy, and far from the "more declarative" approach we wand to take. 

I don't see immediate factorisation, except importing `model` first and then importing base and montant_csg_crds from it, but that's three lines.